### DISCUSSION TITLE
Impossible connection to mosquitto / z2M from a synology nas multi gateways and domoticz all in docker

### DISCUSSION THREAD

**Discussion by yahtala75** [Koenkk/zigbee2mqtt#17323] [General]
Hello,

After running different test, I'm looking for help about my problem.
I am running Eclipse-mosquitto and Zigbee2MQTT and domoticz in docker in a synology nas ds923+.

I have  two routers, one is the ISP router connected to the NAS on LAN 1 and a VPN router (using nordVPN) connected to the NAS on LAN 3.
This is to force the NAS using the VPN to connect on internet and the LAN 1 is to have the hability to connect to the NAS with DNS name from outside.
To be able to connect to the nas from outside using also the vpn, for exemple using the vpn on my phone trying to connect to the nas, I need activate the "Multiple Gateways" option in the synology. But when I activate this option, the mosquitto and Zigbee2MQTT are no more able to connect to the server with domoticz and the "MQTT Client Gateway with LAN Interface" or "MQTT Auto Discovery Clien Gateway with LAN Interface" (Domoticz Error : mosquitto: Failed to start, return code: 14/Connection timed out (Check IP/Port) )

Do you have some ideas to keep the "multiple Gateways" option and let Mosquitto, Zigbe2MQTT and Domoticz work normaly ?

Thank you in advance :)

**Comment by compumark** [comment]
I have exact the same issue. Once "Multiple gateways" is enabled in DSM the MQTT Server is not reachable from Homeassistant anymore - both running on Synology as a separate docker container. Did you found a solution to this issue?
